
 This patch will keep the SD from blocking simultaneous Jobs during
 a tape mount. 

 Apply it to 1.36.0 with:

 cd <bacula-source>
 patch -p0 <1.36.0-sd-block.patch
 make
 ...

Index: src/stored/acquire.c
===================================================================
RCS file: /cvsroot/bacula/bacula/src/stored/acquire.c,v
retrieving revision 1.74
diff -u -r1.74 acquire.c
--- src/stored/acquire.c	16 Oct 2004 11:51:32 -0000	1.74
+++ src/stored/acquire.c	25 Oct 2004 16:12:46 -0000
@@ -339,7 +339,10 @@
 
    if (do_mount || recycle) {
       Dmsg0(190, "Do mount_next_write_vol\n");
-      if (!mount_next_write_volume(dcr, release)) {
+      V(mutex);                       /* don't lock everything during mount */
+      bool mounted = mount_next_write_volume(dcr, release);
+      P(mutex); 		      /* re-lock */
+      if (!mounted) {
 	 if (!job_canceled(jcr)) {
             /* Reduce "noise" -- don't print if job canceled */
             Jmsg(jcr, M_FATAL, 0, _("Could not ready device \"%s\" for append.\n"),
